portfolio/dash/rcx_report_tab.py

```
@@ -0,0 +1,368 @@
+        Input("edge-registry-revision", "data"),
+    )
+    def sync_report_sites(tab, _revision):
+        if tab != "report":
+            return no_update, no_update
+        try:
+            data = _api_get("/api/central/edges")
+            edges = data.get("edges") or []
+            opts = [{"label": f"{e.get('name') or e.get('site_id')} ({e.get('site_id')})", "value": e.get("site_id")} for e in edges]
+            val = opts[0]["value"] if opts else None
+            return opts, val
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)